environments/sandbox.yml

```diff
@@ -126,3 +126,19 @@ cname:
   - name: "cdnverify.bulkscan"
     ttl: 300
     record: "cdnverify.hmcts-bulkscan-shutter-sbox.azureedge.net"
+  - name: “toffee”
+    ttl: 300
+    record: "sdshmcts-sbox.azurefd.net"
+    shutter: false
+  - name: "www.toffee"
```

Comment:
we won't have an ssl cert for this so shouldn't have the www

Reply:
removed both entries (www and afdverfy.www)